aws-auth-cognito/src/main/java/com/amplifyframework/statemachine/codegen/events/SignInChallengeEvent.kt

```diff
@@ -25,6 +25,7 @@ internal class SignInChallengeEvent(val eventType: EventType, override val time:
         data class VerifyChallengeAnswer(val answer: String, val metadata: Map<String, String>) : EventType()
         data class FinalizeSignIn(val accessToken: String) : EventType()
         data class Verified(val id: String = "") : EventType()
+        data class ThrowError(val exception: Exception, val challenge: AuthChallenge) : EventType()
```

Comment:
Have we discussed this fix with iOS? iOS does not have a ThrowError SignInChallengeEvent.

Reply:
Yes I have discussed this with iOS. The ThrowError event in SignInChallengeStateMachine is defined in the State machine definition.